src/Akka.Persistence.MongoDb.Hosting.Tests/Akka.Persistence.MongoDb.Hosting.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The approach was taken to have a separate project for the hosting tests. Can be moved into the main test project if needed.

Reply:
I'd move this into the main test project - no need for a separate project

Reply:
Moved into existing project